### DISCUSSION TITLE
PC engine/TG16 Retrode TG16 adapter

### DISCUSSION THREAD

**Discussion by ztcdeveloper** [sanni/cartreader#335] [Need help]
I am having trouble finding out how to get a hold of this adapter. I see threads mentioning DMing skaman to get one. I understand the adapter converts the HU card slot to plug into the SNES slot for the cart reader to read it. Why is this not a public adapter like the other ones? Would it be possible at least to post the mapping so that I can take a swing at making my own PCB?

**Comment by sanni** [comment]
There was a thread about it on the Retrode forum but that forum seems to be gone now. I have not made an adapter myself yet. The mapping is found in [pinout.ods](https://github.com/sanni/cartreader/blob/master/pinout.ods)

The needed [slot](http://www.tototek.com/store/index.php?main_page=product_info&cPath=22&products_id=210) has not been available for quite some time either.

**Reply by ztcdeveloper** [reply]
Thank you for the quick reply! I am face palming because that file is on the very first page and I somehow missed it.

It's a bummer that the slot isn't available and I've searched forever for one and can't find one. I'm thinking about creative ways to make my own with some combination of 3d printing and some off the shelf parts. But at least with the pinouts I can now start the process of making an adapter one way or another.

If I come up with something (that works) I'll post it here to show off.

**Comment by sanni** [comment]
The slot this import adapter uses looks like a pretty standard PCI slot from a PC mainboard.

![importpce](https://user-images.githubusercontent.com/19169494/155576427-fc0c7337-2f0b-4808-983b-29b33934b88a.jpg)

**Reply by ztcdeveloper** [reply]
Yeah I've also found this PC-Henshin region converter that has the slot on it. But $80 is too pricey just to get a hold of a slot.

http://www.tototek.com/store/index.php?main_page=product_info&products_id=211

![image](https://user-images.githubusercontent.com/43078354/155578554-dfa4c21b-97e1-43d8-a2e3-3c1a29019bed.png)

**Reply by PsyK0p4T** [reply]
Tototek one wasn't perfectly matching the NEC original ones. Its pins got a different spacing.
To make mine I bought a dead PC Engine and removed the HuCard slot from it. A bit long to unsolder, but it's working great !

**Reply by ztcdeveloper** [reply]
That's a bummer! But really good to know! I've thought about doing that myself and may end up doing it if I hit a wall, but it would be great to come up with a solution that is cheap and that it's easy to acquire parts for.

**Comment by sanni** [comment]
This slot is interesting too: https://www.aliexpress.com/item/4000140718173.html

![external-content duckduckgo com](https://user-images.githubusercontent.com/19169494/155612240-9da1d5af-a342-4d68-b85a-f01788ad2f8e.jpg)

Looks like you could maybe file it larger so that the thicker HuCard fits.

**Reply by ztcdeveloper** [reply]
Yeah this does seem like it could work! I'm going to hopefully be placing an order soon to build the full V4 version of the cart reader - so I might grab a couple of these as well to see if they could work at all.

**Reply by tamanegitaro** [reply]
I made one for myself by cutting PCI slot in half but was very very unstable so I couldn’t release the gerber. You can find PC engine to MEGADRIVE/Genesis slot in Japanese amazon market but you need to change the code little bit because it has different pinouts. https://www.amazon.co.jp/dp/B0826P6X3Q/
